addons/crm/models/crm_lead.py

```diff
@@ -387,6 +391,38 @@ def _compute_name(self):
             if not lead.name and lead.partner_id and lead.partner_id.name:
                 lead.name = _("%s's opportunity") % lead.partner_id.name
 
+    @api.depends('partner_id', 'partner_name')
+    def _compute_commercial_partner_id(self):
+        partnered_leads = self.filtered('partner_id')
```

Comment:
Is "partnered" a word :D ?

Reply:
Sure! :)

Though it's not really what it usually means ^^
Renamed `leads_w_partners`